### DISCUSSION TITLE
Why did the hypre parameters I modified fail to take effect？

### DISCUSSION THREAD

**Discussion by Moyiqi** [idaholab/moose#24003] [Q&A General] [answered]
Hello everyone! May I ask a question?

When I modify in moose

-pc_hypre_boomeramg_strong_threshold = 0.7

It outputs
![1680854346343](https://user-images.githubusercontent.com/59696390/230568065-5e4695dd-2955-4163-9c5a-c556d4b9612b.png)
![image](https://user-images.githubusercontent.com/59696390/230568264-5430ed84-fb12-4645-a3d6-782cb12fd792.png)

But the residuals didn't change, did this step not work？

Lily

**Comment by GiudGiud** [comment, marked as answer]
Can you try -pc_hypre_boomeramg_strong_threshold 0.7

it works fine for me

**Reply by Moyiqi** [reply]
Thank you so much!
Lily
